### ISSUE TITLE
Getting panic when using Google Cloud Storage

### ISSUE THREAD

**Issue by shamil** [juicedata/juicesync#99]
version 0.9.3

Getting following error when trying to use GCS storage

```log
/juicefs format --storage gs --bucket gs://bucket-example-test localhost test2
2021/01/20 00:46:45.475761 juicefs[954814] <INFO>: Meta address: redis://localhost
2021/01/20 00:46:45.476692 juicefs[954814] <WARNING>: AOF is not enabled, you may lose data if Redis is not shutdown properly.
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/juicedata/juicesync/object.newGS(0x7ffd6cdcf170, 0x10, 0x0, 0x0, 0x0, 0x0, 0x2, 0x0, 0x16, 0xc00011cd80)
	/go/pkg/mod/github.com/juicedata/juicesync@v0.6.3-0.20210114082630-137546625adb/object/gs.go:145 +0x3e6
github.com/juicedata/juicesync/object.CreateStorage(0x7ffd6cdcf164, 0x2, 0x7ffd6cdcf170, 0x10, 0x0, 0x0, 0x0, 0x0, 0x16, 0xc00011cd80, ...)
	/go/pkg/mod/github.com/juicedata/juicesync@v0.6.3-0.20210114082630-137546625adb/object/object_storage.go:123 +0x1ad
github.com/juicedata/juicefs/pkg/object.CreateStorage(0x7ffd6cdcf164, 0x2, 0x7ffd6cdcf170, 0x10, 0x0, 0x0, 0x0, 0x0, 0xc000559920, 0x45f8c7, ...)
	/go/src/github.com/juicedata/juicefs/pkg/object/interface.go:57 +0x26d
main.createStorage(0xc000559b68, 0x15894c1, 0x8, 0xc00012ba50, 0xe)
	/go/src/github.com/juicedata/juicefs/cmd/format.go:54 +0xa5
main.format(0xc00009b600, 0x8, 0xe)
	/go/src/github.com/juicedata/juicefs/cmd/format.go:165 +0x8fe
github.com/urfave/cli/v2.(*Command).Run(0xc000169440, 0xc00009b380, 0x0, 0x0)
	/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/command.go:163 +0x4ed
github.com/urfave/cli/v2.(*App).RunContext(0xc000243a00, 0x178e700, 0xc0000400b8, 0xc00003c080, 0x8, 0x8, 0x0, 0x0)
	/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:313 +0x81f
github.com/urfave/cli/v2.(*App).Run(...)
	/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:224
main.main()
	/go/src/github.com/juicedata/juicefs/cmd/main.go:74 +0x93c
```

**Comment by davies** [comment]
This should be fixed by #98 , can you update the dependency in juicefs and try again?

```
go get github.com/juicedata/juicesync@master
```

**Comment by davies** [comment]
We have update the juicesync in juicefs, please build from latest main branch, and try again
